### PR TITLE
[RW-6870][risk=no] e2e test take too long to fail during login

### PR DESCRIPTION
**Pull request by aweng98**

### CODE REVIEW

File: e2e/app/page/google-login.ts [hunk 1]
```diff
@@ -171,16 +172,19 @@ export default class GoogleLoginPage {
...
+      await this.page.waitForTimeout(1000);
+    }
+    const emailInputXpath = '//input[@type="email" and @aria-label="Enter recovery email address"]';
+    const [emailInput] = await this.page.$x(emailInputXpath);
+    if (emailInput) {
+      await emailInput.type(config.institutionContactEmail);
       await Promise.all([
-        this.page.waitForNavigation({ waitUntil: ['networkidle2', 'load'], timeout: 30000 }),
-        this.page.keyboard.press(String.fromCharCode(13)), // press Enter key
-        this.page.waitForSelector('app-signed-in', { timeout: 0 })
```
Comment (aweng98) [inline e2e/app/page/google-login.ts:181]: Removed ` {timeout: 0}`.

File: e2e/app/page/base-page.ts [hunk 2]
```diff
@@ -26,7 +26,7 @@ export default abstract class BasePage {
    * Load a URL.
    */
   async gotoUrl(url: string): Promise<void> {
-    await this.page.goto(url, { waitUntil: ['networkidle0', 'load'], timeout: 0 });
+    await this.page.goto(url, { waitUntil: ['networkidle0', 'load'] });
```
[the remark this answers is not in the document]
Comment (calbach) [reply e2e/app/page/base-page.ts:29]: What is the default timeout now for this goto? I would recommend 60s, worst case 90s.

**Review by calbach — APPROVED** [review]
thanks

File: e2e/app/page/base-page.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (aweng98) [reply e2e/app/page/base-page.ts:29]: 60 sec. set [here](https://github.com/all-of-us/workbench/blob/655b9278cb434ed0a126010f2799355b6e3c670e/e2e/jest.test-setup.ts#L23).

**Review by aweng98 — COMMENTED** [review]
Thank you.

**Pull request merged**